lib/helper.py

```diff
@@ -195,6 +195,10 @@ def __init__(self, base_fw=[], opt_fw=[], kvm_fw=[], enable_kvm=False):
         """
         helper class to parse, install, uninstall pip package from user config
         """
+        self.pip_cmd = "pip%s" % sys.version_info[0]
```

Comment:
Here can we exit gracefully attempting to check if python version greater than 3.4?

Reply:
we can do that, we might to explicitly add that in the requirements section for readme aswell, will do it.